src/main/java/com/glencoesoftware/omero/ms/core/PickledSessionConnector.java

```diff
@@ -154,9 +158,13 @@ private void deserializeConnector(Iterator<Op> opIterator) {
                         case "is_public":
                             isPublic = deserializeBooleanField(opIterator);
                             break;
+                        default:
+                            log.warn("Unexpected field name in connector: "
+                                     + fieldName);
                     }
                 } catch (Exception e) {
                     log.error("Exception while deserializing: {}", fieldName);
+                    log.error("Pickled Session: " + b64Session);
```

Comment:
As above.

Also, I would assume we would rather want to surround the `init()` in the constructor with a try block and log this at ERROR level if an exception is thrown there, no?